source/_UIAHandler.py

```diff
@@ -183,6 +183,25 @@ def MTAThreadFunc(self):
 				isUIA8=True
 			except (COMError,WindowsError,NameError):
 				self.clientObject=CoCreateInstance(CUIAutomation._reg_clsid_,interface=IUIAutomation,clsctx=CLSCTX_INPROC_SERVER)
+			# #7345: Instruct UIA to never map MSAA winEvents to UIA propertyChange events.
+			# These events are not needed by NVDA, and they can cause the UI Automation client library to become unresponsive if an application firing winEvents has a slow message pump. 
+			pfm=self.clientObject.proxyFactoryMapping
+			for index in xrange(pfm.count):

```

Comment:
Is the amount of entries in a mapping stable, or can it change while NVDA is running (i.e. can new entries be added for which unregistration has to take place)?